src/app/components/sidebar/filters/Filters.tsx

```diff
@@ -2,28 +2,71 @@ import * as Styled from './Filters.styles';
 import { useIntl } from 'react-intl';
 import { ConditionsPanel } from './condition/ConditionsPanel';
 import { LabelWithButton } from 'ui';
-import { FiltersProps } from './Filters.types';
+import { FiltersProps, ActiveFliterItem } from './Filters.types';
+import { PricePanel } from './pricePanel/PricePanel';
+import { useLocale } from 'hooks';
+import { FormatNumberOptions } from 'react-intl';
+
+export const Filters = ({ onConditionClick, onPriceChange, activeFilters }: FiltersProps) => {
+  const { formatMessage, formatNumber } = useLocale();
+
+  const currencyConfig: FormatNumberOptions = {
+    style: 'currency',
+    currency: 'PLN',
+    maximumFractionDigits: 0,
+  };
+
+  const isAnyFilterApplied =
+    (activeFilters.condition && activeFilters.condition.length > 0) || activeFilters.minPrice || activeFilters.maxPrice;
+
+  const activeFliterItems: ActiveFliterItem[] = [];
```

Comment:
Druga sprawa - może warto byłoby wyciągnąć to jako funkcję pomocniczą do osobnego pliku? Jakiś duży się robi ten plik. Np jako `generateActiveFilterItems` do pliku `Filters.utils.ts`?